spec/factories/document_factory.rb

```diff
@@ -5,19 +5,7 @@
     content_id { SecureRandom.uuid }
     locale { I18n.available_locales.sample }
     title { SecureRandom.alphanumeric(8) }
-    base_path { title ? "#{DocumentTypeSchema.find(document_type).path_prefix}/#{title.parameterize}" : nil }
-    document_type { DocumentTypeSchema.all.reject(&:managed_elsewhere).sample.id }
-
-    trait :with_body_in_schema do
-      document_type do
-        DocumentTypeSchema.all.select { |schema| schema.contents.any? { |field| field.id == "body" } }.sample.id
-      end
-    end
-
-    trait :with_associations_in_schema do
-      document_type do
-        DocumentTypeSchema.all.select { |schema| schema.associations.any? }.sample.id
-      end
-    end
+    base_path { title ? "/prefix/#{title.parameterize}" : nil }
```

Comment:
If we set base_path to a random string, would the tests still pass?

Reply:
We discussed this a bit in person and agreed this is a good compromise that is both random and roughly realistic.